samples/csharp/MicrosoftEduGraphSamples/Workflows/AssignmentWorkflow.cs

```diff
@@ -64,5 +67,95 @@ public async Task<IEnumerable<EducationAssignment>> GetMeAssignmentsFromNonArchi
                 return null;
             }
         }
+        /// <summary>
+        /// Workflow to get assignments evolvable enums
+        /// </summary>
+        /// <param name></param> 
+        public async Task AssignmentsEvolvableEnums()
+        {
+            try
+            {
+                int retries = 0;
+                string assignmentId = string.Empty;
+                string submissionId = string.Empty;
+
+                // Get a Graph client using delegated permissions
+                var graphClient = MicrosoftGraphSDK.GraphClient.GetDelegateClient(_config["tenantId"], _config["appId"], _config["teacherAccount"], _config["password"]);
+
+                // Create a draft assignment with displayName = "Verify assignments states [inactive]"
```

Comment:
revise this comment to: "Create assignment to verify inactive state"

Reply:
resolved